core/components/organisms/table/Header.tsx

```diff
@@ -130,18 +132,19 @@ export const Header = (props: HeaderProps) => {
     if (updateSchema) updateSchema(newSchema);
   };
 
+  const customLabel = customSelectionLabel ? customSelectionLabel : 'item';
```

Comment:
please add items instead item

Reply:
This is already handled inside the code.